### PR TITLE
fix not support python 3.11+

### PR DESCRIPTION
**Pull request by zeroday0619**

### CODE REVIEW

**Review by dimaqq — APPROVED** [review]
LGTM :)

off-topic, why not bump cython up for everyone?
Looking through https://cython.readthedocs.io/en/latest/src/changes.html#id27 I don't see any Python versions being deprecated between 0.29.24..0.29.30, so it should be safe, right?

**Comment by zeroday0619** [comment]
> 
When I checked again, it seems that upgrading the Cython version does not affect the previous version. thank you

**Comment by cclauss** [comment]
Please add `"3.11-dev"` to the testing list so we know this change is effective.
* https://github.com/MagicStack/uvloop/blob/master/.github/workflows/tests.yml#L17

**Review by fantix — APPROVED** [review]

**Comment by fantix** [comment]
Failing because of aio-libs/aiohttp#6600?

**Comment by cclauss** [comment]
#465 is still a problem on Python 3.11 beta 4.

**Comment by zeroday0619** [comment]
Probably... Adding this environment variable will solve the problem.
`AIOHTTP_NO_EXTENSIONS=1`

**Comment by graingert** [comment]
@fantix can you enable this setting so that external contributors can run PRs? https://github.com/MagicStack/uvloop/settings/actions#actions_fork_pr_approvals_first_time_contributor_new_users

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from setuptools.command.sdist import sdist as sdist
 
 
-CYTHON_DEPENDENCY = 'Cython(>=0.29.24,<0.30.0)'
+CYTHON_DEPENDENCY = 'Cython(>=0.29.30,<0.30.0)'
```
Comment (cclauss) [inline setup.py:24]:
```suggestion
CYTHON_DEPENDENCY = 'Cython(>=0.29.32,<0.30.0)'
```
https://pypi.org/project/Cython

**Comment by cclauss** [comment]
Some progress... https://github.com/aio-libs/aiohttp/issues/6600#issuecomment-1203655241

**Comment by cclauss** [comment]
I believe that for now experimentation on Py311 will require `AIOHTTP_NO_EXTENSIONS=1` and `--no-binary`.
* https://github.com/aio-libs/aiohttp/issues/6600#issuecomment-1229417344

**Comment by zeroday0619** [comment]
> I believe that for now experimentation on Py311 will require `AIOHTTP_NO_EXTENSIONS=1` and `--no-binary`.
> * https://github.com/aio-libs/aiohttp/issues/6600#issuecomment-1229417344

Can I add it to test code?

**Comment by fantix** [comment]
> > I believe that for now experimentation on Py311 will require `AIOHTTP_NO_EXTENSIONS=1` and `--no-binary`.
> > 
> > * [Fails to build on Python 3.11 - longintrepr.h: No such file or directory aio-libs/aiohttp#6600 (comment)](https://github.com/aio-libs/aiohttp/issues/6600#issuecomment-1229417344)
> 
> Can I add it to test code?

Let's just skip aiohttp in 3.11 for now as #488 does until it supports 3.11. I'll merge in and commit this.

**Comment by Kludex** [comment]
Thanks @fantix 😁🙏

**Comment by finswimmer** [comment]
:rocket: Can't wait for a new release :smiley:

**Comment by ahopkins** [comment]
Sweet! 👏

**Comment by KevinRodriguez777** [comment]
excited for this!!

**Pull request merged**